### ISSUE TITLE
[Bug]: Not possible to use syslog for both normal logs and audit logs with different tags

### ISSUE THREAD

**Issue by Ivansss** [nextcloud/server#49831]
Reading https://www.php.net/manual/en/function.openlog.php , it’s not possible to keep 2 connections open to syslog with a different tag.
So this is a limitation of our log system, it’s not possible to use syslog for both normal logs and audit logs with different tags.

Not sure if it is possible to fix this without losing performance, but this limitation could be explained in our logging documentation.
https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/logging_configuration.html

**Comment by susnux** [comment]
> it’s not possible to keep 2 connections open to syslog with a different tag.

Thats true but it is not a performance issue to reopen the logger with a different tag.
